adaptto-2018-example.yaml

```diff
@@ -2,21 +2,24 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   labels:
-    run: example-kubernetes
-  name: example-kubernetes
+    app: adaptto-2018-example
+    ver: 1.0.0-SNAPSHOT
+  name: adaptto-2018-example
 spec:
+  replicas: 2
   selector:
     matchLabels:
-      run: example-kubernetes
+      app: adaptto-2018-example
   template:
     metadata:
       labels:
-        run: example-kubernetes
+        app: adaptto-2018-example
     spec:
       terminationGracePeriodSeconds: 30
       containers:
-      - image: cschneider/osgi.example.kubernetes
-        name: example-kubernetes
+      - image: dockeradulvac.azurecr.io/osgi.example.kubernetes
```

Comment:
Why do you use a special docker repo here. I would like people to be able to install the example on their own. 
The docker image from my account should at least be publicly available.
cschneider of course is also not ideal but at least the repo there is public. So it would be ideal if we find a way so someone trying the example can easily use his own docker hub account.

Reply:
You're right, it could be public, but we indeed need to make it configurable.

Reply:
Any idea how to do that? Does kubernetes maybe allow to use placeholders from env or similar?

Reply:
well... helm for templating :/

Reply:
Valid .. but I guess a bit oversized for what we do :-)